### ISSUE TITLE
About ORB-v3 training config

### ISSUE THREAD

**Issue by KIKO-9** [orbital-materials/orb-models#110]
Hi, orb team,I've read orb series papers, amazing work! But I do not find any Training Config for Orbv3 model(e.g., epochs, loss for OMat and sAlex+MPtrj), are there any plans to release these Config?

**Comment by vsimkus** [comment]
Hi, thanks for the interest! Unfortunately the public repository currently does not have the training code because it heavily relies on internal-only libraries. Sorry about that!
